src/arrayMethodSort.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) > String(b) ? 1 : 0) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(
    compareFunction = (a, b) => String(a) > String(b)
  ) {
```
I'm not sure that writing a function directly in parameters list is a good idea. The readability of this is pretty bad.
You can make a small nested comparator function and use it as a default. But it's up to you.

Also, `true` is greater than `0`.